### DISCUSSION TITLE
Does this work with markdown?

### DISCUSSION THREAD

**Discussion by Cormanz** [jacwright/typewriter#71] [Q&A]
Hi, I'm working on a project in Svelte and I was wondering if TypeWriter could take in markdown text to start with, and output markdown text too.

**Comment by taylorhadden** [comment]
Hi Cormanz. I am _also_ using typewriter in a project in Svelte using markdown. My editing model is somewhat like [Typora](https://typora.io), where the formatting characters are visible when the cursor is in them, but hidden otherwise. This has required some custom modules and typesets, but it's worked out great.

If what you want to achieve is more like using markdown as the source file while using Typora's default typesets, you should be able to do this rather easily. There is no baked in solution that I'm aware of (@jacwright, correct me if I'm wrong), but you would just need to translate the markdown file to the annotated `delta` data that Typewriter expects. When saving, you would then do that process in reverse.

I hope that helps, and I hope the answer was somewhat useful even if it's three months later.
